app/ai/voice/agents/breeze_buddy/agent/__init__.py

```
@@ -707,6 +713,47 @@ async def _run_with_tracing(self, runner: PipelineRunner) -> None:
+                merchant_id=merchant_id,
+                template_id=template_id,
+                azure_embedding_deployment=RAG_EMBEDDING_DEPLOYMENT,
+            )
+            await self._rag_router.start()
+            self._rag_context_processor = RagContextProcessor(self._rag_router)
+            logger.info(
+                "RAG initialised: gs://.../%s/%s",
+                merchant_id,
+                template_id,
+            )
+        except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)